tests/unit/store/mutations.spec.ts

```diff
@@ -9,6 +9,7 @@ describe( 'mutations', () => {
 			conditionRow: {
 				valueData: { value: 'foo' },
 				propertyData: { id: 'P123', label: 'abc' },
+				propertyValueRelationData: { value: 'with' },
```

Comment:
I think the value should be = 'matching' for clarity or enum `propertyValueRelation: {
					value: PropertyValueRelation.Matching,
				},`

Reply:
updated